improver/calibration/simple_bias_correction.py

```diff
@@ -302,6 +353,10 @@ def _check_forecast_bias_consistent(
         """Check that forecast and bias values are defined over the same
         valid-hour and forecast-period.
 
+        Checks that between the bias_data Cubes there is a single coordinate forecast_reference_time
+        and single coordinate forecast_period. Then check forecast Cube contains the same single
+        coordinate forecast_reference_time and single coordinate forecast_period.
+
```

Comment:
The check on the `forecast_reference_time` coordinate is to ensure that the bias_cubes, and the forecast cube all have common cycle-hour. Maybe this could replaced with the following:
```suggestion
        Checks that between the bias_data Cubes there is a common hour value for the forecast_reference_time
        and single coordinate value for forecast_period. Then check forecast Cube contains the same hour value
        for the forecast_reference_time and the same forecast_period coordinate value.
```